examples/CountryExample.php

```diff
@@ -9,6 +9,11 @@ class CountryExample extends Model
 {
 	use \Sushi\Sushi;
 
+    /**
+     * Countries rows.
+     *
+     * @var array $rows
```

Comment:
Laravel docblocks style does not display variable names - this should be `@var array`